data/json/items/comestibles/med.json

```diff
@@ -0,0 +1,251 @@
+[
+    {
+        "type" : "COMESTIBLE",
+        "id" : "cotton_ball",
+        "name" : "cotton balls",
+        "name_plural" : "cotton balls",
+        "category" : "spare_parts",
+        "weight" : 180,
+        "color" : "white",
+        "container" : "bag_plastic",
+        "comestible_type" : "MED",
+        "symbol" : "*",
+        "quench" : 0,
```

Comment:
Sounds crude but I was just going to round up all the different "comestible_type" : "MED", items and then get all the unnecessary entries at the end all at once.